internal/app/skuba/node/validate/validate.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package validate
+
+import (
+	"fmt"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+)
+
+const (
```

Comment:
Code-lint is complaining on this, please change it to
```
const ()
```

Reply:
I will just remove that completely. :man_facepalming: that I left that here.